resources/views/email/footer.blade.php

```diff
@@ -10,6 +10,12 @@
                     <img src="{{ $message->embed(public_path().'/images/eia-bsqr.png') }}" alt="PayBySquare" class="byqr" />
                   </td>
                 </tr>
+              <tr>
+                  <td class="content-block">
+                      <p><strong>Tento email dostávate, lebo ste sa prihlásili na odber upozornení na webe <a href="https://eia.cyklokoalicia.sk">Sleduj EIA</a>.</strong></p>
+                      <p><a href="{{ $project->unsubscribelinkloc }}?all=true" title="Zrušenie odberu upozornení pre všetky lokality">Odhlásenie z odberu upozornení pre všetky lokality</a>.</p>
```

Comment:
@pato1 Je dobre navazovat "unsubscribe all" na konkretny projekt? Pride mi to ako nieco strasne zle maintainovatelne do buducna. Nakolko je zlozite spravit globalny endpoint? Teda aby nebol naviazany na projekt.

Reply:
Ok. Uz chapem.